scripts/jobrunner.sh

```diff
@@ -71,6 +70,14 @@ done
 chown -R jobrunner:jobrunner "$HIGH_PRIVACY_STORAGE_BASE"
 chown -R jobrunner:jobrunner "$MEDIUM_PRIVACY_STORAGE_BASE"
 
+# ensure docker images present
+# Note: does not update, as that is currently done manually
+
+for image in cohortextractor python jupyter r base-docker busybox; do
```

Comment:
Not a point for now, but I wonder if we should think about moving the list of allowed images out of the job-runner Python code and into some easily readable config file so we can reuse it here.